packages/core/src/messages/responses/enums/store/scalar/index.ts

```diff
@@ -0,0 +1,17 @@
+export enum StoreGetResponse {
```

Comment:
```suggestion
export enum StorageGetResponse {
```

We're favoring `Storage` over `Store` for naming in the data plane operations.